document.txt

```diff
@@ -0,0 +1,8 @@
+This is an important document 
+-------------
+
+Here's an important list:

```

Comment:
reply works

---
In reply to: [56726622](https://github.com/kivancmuslu/CFTest/pull/2#discussion_r56726622) [](ancestors = 56726622)